system-variables.md

```diff
@@ -282,6 +282,13 @@ This variable is an alias for `last_insert_id`.
 >
 > Unlike in MySQL, the `max_execution_time` system variable currently works on all kinds of statements in TiDB, not only restricted to the `SELECT` statement. The precision of the timeout value is roughly 100ms. This means the statement might not be terminated in accurate milliseconds as you specify.
 
+### placement_checks
+
+- Scope: SESSION | GLOBAL
+- Default value: ON
+- This variable controls whether DDL validates that [Placement Rules in SQL](/placement-rules-in-sql.md) are valid when creating tables.
```

Comment:
Not sure whether my comment is correct. PTAL
```suggestion
- This variable controls whether a DDL execution validates [Placement Rules in SQL](/placement-rules-in-sql.md) when a table is created.
```

Reply:
How about "ddl statements" instead of DDL execution?

Reply:
Modified.

Reply:
OK. I've made further small edits (DDL statements implies "when creating tables", since that is a DDL statement).